### PR TITLE
Add ColorService and enableBrowserColor method

### PR DESCRIPTION
**Pull request by scote**
See the angular material documentations : 
- https://material.angularjs.org/latest/api/service/$mdColors
- https://material.angularjs.org/latest/api/service/$mdThemingProvider

### CODE REVIEW

File: angular-material/angular-material.d.ts [hunk 1]
```diff
@@ -73,6 +73,12 @@ declare namespace angular.material {
         initialValue(initialValue: string): IPromptDialog;
     }
 
+    interface IColorService {
+        applyThemeColors(element:Element|JQuery, colorExpression: Object): void;
+        getThemeColor(expression: Object): string;
```
Comment (andy-ms) [inline angular-material/angular-material.d.ts]: The documentation you provided seems to indicate that `expression` should be a `string`.

File: angular-material/angular-material.d.ts [hunk 2]
```diff
@@ -73,6 +73,12 @@ declare namespace angular.material {
         initialValue(initialValue: string): IPromptDialog;
     }
 
+    interface IColorService {
+        applyThemeColors(element:Element|JQuery, colorExpression: Object): void;
```
Comment (andy-ms) [inline angular-material/angular-material.d.ts]: According to the docs, `colorExpression` is a `json object, keys are css properties and values are string of the wanted color, for example: {color: 'red-A200-0.3'}.` The type could be `{ [cssPropertyName: string]: string }`.

**Comment by andy-ms** [comment]
Thanks. I'm marking this as "Awaiting reviewer feedback" in case one of the pinged authors has something to say. If they don't respond, ping me after 3 days and I'll merge.

**Comment by AlStar01** [comment]
:+1:

**Pull request merged**